apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -9816,6 +9816,12 @@ body.update-scroll-hidden {
         width: auto;
         max-width: 300px;
     }
+    .d-sm-inline {
+        display: inline-block !important;
+    }
```

Comment:
by-default class d-inline-block is used for display:inline-block thing, please use that. d-sm-inline is used for other css. @ravinderwalkover

Reply:
Please check the logic also behind this not just name.